app/views/admin/data_imports/show.html.erb

```diff
@@ -13,36 +13,49 @@
           )
         end %>
 
+    <%= content_for(:header_middle) %>
+
     <%= if accessible_action?(page.resource, :destroy)
-          link_to(
+          button_to(
```

Comment:
This is a security update introduced because methods that make changes to the server are better handled by form submissions ([see the reference here](https://github.com/thoughtbot/administrate/pull/2516)).